app/views/candidate_mailer/find_has_opened.erb

```diff
@@ -0,0 +1,20 @@
+<% if @application_form.first_name.present? %>
+  Dear <%= @application_form.first_name %>,
+<% end %>
+
```

Comment:
Move the newline within the condition?

```suggestion
<% if @application_form.first_name.present? %>
  Dear <%= @application_form.first_name %>,

<% end %>
```

Reply:
The way that the formatting works means that it won't ever start with a newline anyway so we're all good